SU-SWS/stanford_profile_helper
# D8CORE-3163: adding a little space on lists in columns

# READY FOR REVIEW

# Summary
- Adding a bit of padding on cards and lists when they are in columns
- Fix for a bug I found when merging. on D8CORE-3164, I didn't account for the news list in 12-of-12 columns. So adjust that selector.

# Needed By (Date)
- 1/29

# Urgency
- important

# Steps to Test

1. Pull in this change. Clear caches.
2. Create a page with lists of news and events and people.
3. Put them in columns and as filling the paragraph row.
4. Notice the columns indent the card/list a little on each side but not the intro paragraph.

# Affected Projects or Products
- Stanford Basic theme

# Associated Issues and/or People
- D8CORE-3163
- D8CORE-3164 -fix
- https://github.com/SU-SWS/stanford_profile_helper/pull/64
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
